metricbeat/docker-compose.yml

```diff
@@ -15,11 +15,11 @@ services:
 
   # Used by base tests
   elasticsearch:
-    image: docker.elastic.co/integrations-ci/beats-elasticsearch:${ELASTICSEARCH_VERSION:-7.7.0}-1
+    image: docker.elastic.co/integrations-ci/beats-elasticsearch:${ELASTICSEARCH_VERSION:-7.9.0}-1
     build:
       context: ./module/elasticsearch/_meta
       args:
-        ELASTICSEARCH_VERSION: ${ELASTICSEARCH_VERSION:-7.7.0}
+        ELASTICSEARCH_VERSION: ${ELASTICSEARCH_VERSION:-7.9.0}
```

Comment:
It seems there are some updates to 7.9.0 instead of 7.9.2. Is that intentional?

Reply:
yep, it is intentional, it is based on previous PRs and how the stack is updated, maybe @andrewkroh or @urso can give us more info why it is in that way of if its wrong. For me it is weird.